### DISCUSSION TITLE
How do i add sitting function to object

### DISCUSSION THREAD

**Discussion by Souza-R** [acemod/ACE3#8894] [Q&A]
Hey guys, how do i add a sitting function to a custom wooden bench?

**Comment by Drofseh** [comment]
https://ace3.acemod.org/wiki/framework/sitting-framework.html

**Reply by Souza-R** [reply]
but is there any way to call the function add sitting inside the object init ( 3DEN editor )?

**Reply by Drofseh** [reply]
No, it needs the sitting positions defined in the object's config.
https://github.com/acemod/ACE3/blob/master/addons/sitting/functions/fnc_addSitActions.sqf#L27
